src/main/java/de/tum/in/www1/artemis/domain/Course.java

```diff
@@ -191,6 +193,9 @@ public class Course extends DomainObject {
     @JsonIgnoreProperties("course")
     private Set<Organization> organizations = new HashSet<>();
 
+    @Transient
+    private List<PlagiarismCase> plagiarismCasesTransient;
```

Comment:
Is this really needed? we should avoid spamming too many transient values into our domain objects